apps/meteor/app/livechat/server/api/v1/visitor.ts

```diff
@@ -29,14 +30,13 @@ API.v1.addRoute('livechat/visitor', {
 		});
 
 		const { token, customFields } = this.bodyParams.visitor;
-		const guest: ILivechatVisitorDTO = { ...this.bodyParams.visitor };
+		const guest = {
+			...this.bodyParams.visitor,
+			phone: this.bodyParams.visitor.phone ? { number: this.bodyParams.visitor.phone as string } : undefined,
```

Comment:
Why?

If `phone` accept undefined, why is the ternary needed?

Reply:
worst case, something like: `typeof xxx.phone === 'string' ? xxx.phone : undefined` to avoid the unsafe cast